common/lib/api-client/models.js

```diff
@@ -52,6 +52,8 @@ function defineModels (jsonApi) {
   })
 
   jsonApi.define('assessment_question', {
+    created_at: '',
+    updated_at: '',
```

Comment:
We don't have `updated_at` (yet) - we don't track changes, we just overwrite what's there. We do have `expires_at` (which is empty by default).